docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -0,0 +1,334 @@
+
+<br/>
+
+## `connect`
+
+Using `connect` in the body of a request performs a partial update, adding specified relations.
+
+`connect` accepts either a shorthand or a longhand syntax.
+
+| Syntax type | Syntax example |
+| ------------|----------------|
+| shorthand   | `connect: [2, 4]`
+| longhand    | ```connect: [ {id: 2}, {id: 4} ]``` |
+
+You can also use the longhand syntax to [reorder relations](#relations-reordering).
+
+`connect` can be used in combination with [`disconnect`](#disconnect).
+
+<br />
+
+
+:::::: tabs card
+
+::::: tab Shorthand syntax example
+
+::: request Example request using the shorthand syntax
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: { 
+      connect: [2, 4],
+    }
+  }
+}
+```
+
+:::
+
+:::::
+
+::::: tab Longhand syntax example
+
+::: request Example request using the longhand syntax
+
+`PUT http://localhost:1337/api/restaurants/1`
+
+```json
+{
+  data: {
+    categories: { 
+      connect: [
+        { id: 2 },
+        { id: 4 }
+      ], 
+    }
+  }
+}
+```
+
+:::
+
+:::::
+::::::
+
+
+### Relations reordering
+
+Positional arguments can be passed to the longhand syntax of `connect` to define the order of relations.
+
+The longhand syntax accepts an array of objects, each object containing the `id` of the relation to be connected and an optional `position` object to define where to connect the relation.
+
+::: note Different syntaxes for different relations
+The syntaxes described in this documentation are useful for one-to-many, many-to-many and many-ways relations.<br />For one-to-one, many-to-one and one-way relations, the syntaxes are also supported but only the last relation will be used, so it's preferable to use a shorter format (e.g.: `{ data: { category: 2 } }`, see [REST API documentation](/developer-docs/latest/developer-resources/database-apis-reference/rest-api.md#requests)).
+:::
+
+To define the `position` for a relation, pass one of the following 4 different positional attributes:
+
+| Parameter name and syntax | Description                                                            | Type       |
+| ------------------------- | ---------------------------------------------------------------------- | ---------- |
+| `before: id`              | Positions the relation before the given `id`.                          | Entry `id` |
+| `after: id`               | Positions the relation after the given `id`.                           | Entry `id` |
+| `start: true`             | Positions the relation at the start of the existing list of relations. | Boolean    |
+| `end: true`               | Positions the relation at the end of the existing list of relations.   | Boolean    |
+
+The `position` argument is optional and defaults to `position: { end: true }`.
+
+:::note Sequential order
+Since `connect` is an array, the order of operations is important as they will be treated sequentially (see combined example below).
+:::
+
+::::: tabs card
```

Comment:
I think the database record examples are the same for each example? If so, it might help to put them above the tabs card.

Reply:
Interesting. Indeed these are the same, but I designed the tabs' content to be read independently and from start to finish. I thought it could be confusing to have the "context" of an example outside the example 🤔 WDYT?